client/app/account/user/agreements/user-agreements.service.js

```diff
@@ -1,19 +1,84 @@
 angular.module("UserAccount").service("UserAccount.services.agreements", [
     "$http",
     "$q",
+    "$translate",
     "UserAccount.constants",
     "$cacheFactory",
-    function ($http, $q, constants, cache) {
+    function ($http, $q, $translate, constants, cache) {
         "use strict";
 
         const userAgreementsCache = cache("USER_AGREEMENTS");
 
         const proxyPath = `${constants.swsProxyRootPath}me`;
 
+        const GDPR_AGREEMENTS_INFOS = [
+            {
+                agreementId: 1800,
```

Comment:
Why do we need to put an `agreementId` in the front-end? :(

Should be managed by API :)